alacritty.yml

```diff
@@ -231,6 +231,15 @@ mouse:
   # To disable this completely, set `faux_scrollback_lines` to 0.
   faux_scrollback_lines: 1
 
+  # Ignore first click in unfocused wdinow.
+  #
+  # The `ignore_unfocused_click` setting controls whether or not an
+  # unfocused terminal will respond to the mouse click that causes
+  # it to gain focus.  On MacOS this defaults to true, and on other
```

Comment:
Super tiny nitpick, but there's an extra ` ` before the `On MacOS` here.

Reply:
LOL.  I should have looked at the style of the rest of the file and followed that.  Fixed, thanks.